### ISSUE TITLE
Specify types and default values in the DATA section

### ISSUE THREAD

**Issue by osuwariboy** [HcySunYang/vuese#87]
Hi there, I like the plugin, but I'm simply wondering if it's possible to do more than add a description to elements located in the "data" section of a component. When in the props section, you can be very descriptive with your entries thanks to Vuejs syntax. The "data" section on the other doesn't have that... so is there some sort of mechanism to supplement things here?

**Comment by HcySunYang** [comment]
Unfortunately, I don't have a better idea, except for adding extra annotations.

**Comment by osuwariboy** [comment]
Yeah, that was more of a "feature request", but I couldn't put the correct label

**Comment by IWANABETHATGUY** [comment]
could you please give some code sample, follow the issue template, give the behavior you expect
